src/main/window-manager.ts

```diff
@@ -41,12 +41,10 @@ export class WindowManager extends Singleton {
 
   private async initMainWindow(showSplash: boolean): Promise<BrowserWindow> {
     // Manage main window size and position with state persistence
-    if (!this.windowState) {
-      this.windowState = windowStateKeeper({
-        defaultHeight: 900,
-        defaultWidth: 1440,
-      });
-    }
+    this.windowState ??= windowStateKeeper({
+      defaultHeight: 900,
+      defaultWidth: 1440,
+    });
```

Comment:
Better commit message would help understanding this.

Reply:
It literally is just a modernization